### ISSUE TITLE
Add option to stop resizing/centering view when adding route

### ISSUE THREAD

**Issue by Archy-X** [markusenglund/great-circle-map#22]
An option to stop the screen from resizing and centering when adding a route would be helpful. When you are focusing on one area of the map, the resizing makes it hard to see the short route you just added if you have many long routes that the window resizes to fit.

**Comment by markusenglund** [comment]
True, I'll try to add it when I get the time to start developing this project again. Hopefully sometime in the next half year.
